subsys/usb/device/class/cdc_acm.c

```diff
@@ -119,6 +119,10 @@ struct cdc_acm_dev_data_t {
 	bool configured;
 	/* CDC ACM suspended flag */
 	bool suspended;
+	/* CDC ACM paused flag */
+	bool receive_paused;
```

Comment:
s/receive_paused/rx_paused/g

Reply:
@jfischer-no Could you explain what you mean by this comment?

Reply:
https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html